utils/build-script

```diff
@@ -512,11 +512,13 @@ also build for Apple watchos, but disallow tests that require an watchOS device"
         action="store_true")
 
     parser.add_argument("--swift-analyze-code-coverage",
-        help="enable code coverage analysis in Swift",
-        action="store_const",
-        const=True,
-        dest="swift_analyze_code_coverage",
-        default=False)
+        help="""enable code coverage analysis in Swift (set to `merged` to merge
+and remove intermediary profdata files)""",
+        const="not-merged",
+        choices=["false", "not-merged", "merged"],
+        nargs='?',
```

Comment:
Could we always require the option value?  This is an expert-only option and adding subtle implicit behavior only adds debugging time.